docs/developer-docs/integrations/sns/tokenomics/preparation.md

```diff
@@ -62,6 +58,18 @@ ledger canister such as the token name, token symbol, and the ledger transaction
       swap tokens are reserved for future swaps (separate from the treasury). 
       Note that future swap are not yet designed, but having the tokens reserved
       makes SNSs forward compatible to such a feature if it is added in the future.
+
+<!-- NEW BULLET POINT:
+**Decentralization swap parameters**: this defines the parameters of the decentralization swap. MAKE SUB-BULLETS
+**Rules of the swap** such as the minimum number of ICP tokens that the swap must collect to be successful and the maximum number of ICP tokens that it will collect. 
+**Conditions of swap participation** such as the definition of a text that must be confirmed by users when participating in the swap and the possibility to define countries from which users cannot participate (by IP addresses).
+**Fallback controllers of the dapp:** Another important parameter is the set of fallback controllers of the dapp.
```

Comment:
@DanielThurau could you please confirm that this is still true with 1proposal?

Reply:
I'm not entirely sure what I'm confirming among all of these bullet points but I read them all and nothing in them is false...so confirmed :)